crates/astria-sequencer/src/transaction/checks.rs

```diff
@@ -25,32 +25,11 @@ use tracing::instrument;
 
 use crate::{
     accounts::StateReadExt as _,
-    address::StateReadExt as _,
     app::StateReadExt as _,
     bridge::StateReadExt as _,
     ibc::StateReadExt as _,
 };
 
-#[instrument(skip_all)]
-pub(crate) async fn check_nonce_mempool<S: StateRead>(
```

Comment:
I'd like for all of these to migrate over to the mempool at some point.

Reply:
created an issue https://github.com/astriaorg/astria/issues/1632